<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D_Viewer</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background: QLinearGradient(x0:1, y0: 1, 
                            x0:2,  y2:1, 
                            stop: 0   rgba(255, 221, 190, 255), 
                            stop: 0.3 rgba(255, 221, 170, 200), 
                            stop: 1   rgba(255, 221, 140, 180));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="fileInfoTop">
       <item>
        <widget class="QPushButton" name="fileDialogButton">
         <property name="font">
          <font>
           <family>URW Bookman</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: rgb(230, 97, 0);
color: white; 
border: 1px solid gray;
}
QPushButton:pressed {
background-color: rgb(165, 29, 45);
}</string>
         </property>
         <property name="text">
          <string>Open file</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="fileNameLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="vLable">
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>URW Bookman</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>Vertices:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="vertices_count">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="eLabel">
         <property name="font">
          <font>
           <family>URW Bookman</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0)</string>
         </property>
         <property name="text">
          <string>Edges:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="edges_count">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>561</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="openGLlayout"/>
   </widget>
   <widget class="QPushButton" name="none_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>300</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(1, 105, 61);
color: white; 
border: 1px solid gray;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QPushButton" name="circle_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>340</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(1, 105, 61);
color: white; 
border: 1px solid gray;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="square_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>380</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(1, 105, 61);
color: white; 
border: 1px solid gray;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Vertices type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parallel_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(1, 105, 61);
color: white; 
border: 1px solid gray;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>Parallel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="central_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>80</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(1, 105, 61);
color: white; 
border: 1px solid gray;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>Central</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Projection type</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_move_y">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_move_z">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_move_x">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_scale">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gif_button">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>580</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(1, 105, 61);
	color: white; 
	border: 1px solid gray;
	border-radius: 15px;
}

QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>GIF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_rotate_x">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_rotate_y">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_rotate_z">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="filename_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>561</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="background_color">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>450</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(1, 105, 61);
color: white; 
border: 1px solid gray;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>Background</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bmp_button">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>580</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(1, 105, 61);
	color: white; 
	border: 1px solid gray;
	border-radius: 15px;
}

QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>BMP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jpeg_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>580</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(1, 105, 61);
	color: white; 
	border: 1px solid gray;
	border-radius: 15px;
}

QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>JPEG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Edges type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solid_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>170</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(1, 105, 61);
color: white; 
border: 1px solid gray;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>Solid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dashed_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>210</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(1, 105, 61);
color: white; 
border: 1px solid gray;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>Dashed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>420</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edges_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>480</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(1, 105, 61);
color: white; 
border: 1px solid gray;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>Edges</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertices_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>510</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(1, 105, 61);
color: white; 
border: 1px solid gray;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(38, 162, 105);
}</string>
    </property>
    <property name="text">
     <string>Vertices</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_vertices">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>530</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_edges">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>470</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>Edges thickness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>Vertices size</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
